JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -88,12 +279,28 @@ public Integer call() throws Exception {
             System.out.println(" Check the login credentials and try again.");
         } catch (SQLException e) {
             e.printStackTrace();
+        }finally {
+		//TearDown();
         }
         return 0;
     }
-
+    
+    // load configuration properties
+    public static void loadProperties()
+    {
+	    prop = new Properties();
+		try (InputStream input = new FileInputStream("src/main/java/config.properties")) {
```

Comment:
You need to upload the config.properties file with empty values. Be very careful that you don't upload any client secrets.

Reply:
I have not uploaded the config,properties file yet. I will upload the file with empty values/demoValues.

Reply:
Will upload config.properties file with empty values.